app/contributor-preferences.html

```diff
@@ -411,6 +411,11 @@
         <input type="checkbox" class="enabletype" value="isFirstRun" id="showWelcomeWindow">
         <label data-i18next="contributorPreferences.showWelcomeWindow" for="showWelcomeWindow"></label>
       </div>
+      <div>
+        <input type="checkbox" class="enabletype" value="actAsRegularWindow" id="actAsRegularWindow">
+        <label data-i18next="contributorPreferences.actAsRegularWindow" for="actAsRegularWindow"></label>
+      </div>
+      <div></div>
```

Comment:
Let's make changes here: 
![image](https://user-images.githubusercontent.com/4745859/109636746-3f05ae80-7b4c-11eb-8ceb-bed39a0456d0.png)

1. Let's do it similar to this: 
![image](https://user-images.githubusercontent.com/4745859/109636831-5b095000-7b4c-11eb-8acc-49932cddfd13.png)

For example: 
```
Act as regular window :  When enabled, Break window will become focusable, it won't be 'Always on top' anymore and it will become available in taskbar. 

(checkbox) Show breaks as regular windows
``` 

You are also missing translations. You only need to add English ones. 

You can test this window by changing `isContributor: false` to `isContributor: true` in `main.js`

Reply:
changed in https://github.com/hovancik/stretchly/pull/875/commits/404d234471252d8e005ec6bae18c706e599efd81

![image](https://user-images.githubusercontent.com/9987548/109698776-70a46700-7b98-11eb-8ae7-a73a64455fb1.png)

Reply:
Looks good! 

Small issue with next block; `Appearance` should be on left side

![image](https://user-images.githubusercontent.com/4745859/109813883-32598700-7c2e-11eb-8bf4-4eb26508c9f4.png)

Reply:
Don't know how I missed that!
Fixed in https://github.com/hovancik/stretchly/pull/875/commits/b31b5a70433819498f2ba88fc6f124807d64dfca
![image](https://user-images.githubusercontent.com/9987548/109825732-c46b8a80-7c42-11eb-9dbd-f52e9858f79f.png)